extensions/undertow/runtime/src/main/java/io/quarkus/undertow/runtime/QuarkusExceptionHandler.java

```diff
@@ -0,0 +1,61 @@
+package io.quarkus.undertow.runtime;
+
+import java.io.IOException;
+import java.util.UUID;
+
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+
+import org.jboss.logging.BasicLogger;
+import org.jboss.logging.Logger;
+
+import io.undertow.UndertowLogger;
+import io.undertow.server.HttpServerExchange;
+import io.undertow.servlet.ExceptionLog;
+import io.undertow.servlet.api.ExceptionHandler;
+import io.undertow.util.AttachmentKey;
+
+public class QuarkusExceptionHandler implements ExceptionHandler {
+
+    static AttachmentKey<String> ERROR_UUID = AttachmentKey.create(String.class);
+
+    @Override
+    public boolean handleThrowable(HttpServerExchange exchange, ServletRequest request, ServletResponse response, Throwable t) {
+        String uid = UUID.randomUUID().toString();
```

Comment:
Aren't random UUIDs a bit heavy to generate (because they use `SecureRandom`)?  Maybe we'd want something sequential instead?